### DISCUSSION TITLE
range slider

### DISCUSSION THREAD

**Discussion by AsutoshBaral** [filamentphp/filament#8875] [Help]

**Comment by leandrocfe** [comment]
[custom-field-classes](https://filamentphp.com/docs/3.x/forms/fields/custom#custom-field-classes)

```php
RangeSlider::make('slider')
```
```html
<!-- resources/views/forms/components/range-slider.blade.php -->
<x-dynamic-component :component="$getFieldWrapperView()" :field="$field">
    <input type="range" {{ $applyStateBindingModifiers('wire:model') }}="{{ $getStatePath() }}">
</x-dynamic-component>
```

[build-advanced-components-for-filament](https://laracasts.com/series/build-advanced-components-for-filament)

**Reply by ziming** [reply]
like to ask, does $applyStateBindingModifiers send network request to the server all the time or not?

My guess is no, but like to confirm
